frontend/web/components/base/forms/Tabs.js

```diff
@@ -79,14 +79,14 @@ const Tabs = class extends React.Component {
                     }
                     this.props.onChange?.(i)
                   }}
-                  className={`btn-tab ${isSelected ? ' tab-active' : ''}`}
+                  className={`btn-tab ${this.props.noFocus ? 'btn-no-focus' : ''} ${isSelected ? ' tab-active' : ''}`}
```

Comment:
This is a very specific implementation related to the hook. The selected tab would keep the focus on after interacting with the newly implemented modal